ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -604,12 +600,11 @@ getCompletions plId ideOpts CC {allModNamesAsNS, anyQualCompls, unqualCompls, qu
         , enteredQual `T.isPrefixOf` label
         ]
 
-      filtImportCompls = filtListWith (mkImportCompl enteredQual) importableModules
+      filtImportCompls = filtListWith (mkImportCompl enteredQual) $ map fst $ HM.toList moduleExportsMap
```

Comment:
You could use [HM.keys](https://hackage.haskell.org/package/unordered-containers-0.2.14.0/docs/Data-HashMap-Strict.html#v:keys) here to simplify

Reply:
addressed